frontend/src/static/css/tailwind.css

```
@@ -533,6 +539,158 @@
+	--text-link: var(--cinemata-sunset-horizon-200);
+	--text-link-hover: var(--cinemata-sunset-horizon-100);
+	--text-success: var(--cinemata-green-500);
+	--text-warning: var(--cinemata-amber-500);
+
+	--border-default: var(--cinemata-pacific-deep-700);
+	--border-subtle: var(--cinemata-pacific-deep-800);
+	--border-strong: var(--cinemata-pacific-deep-400);
+	--border-input: var(--cinemata-sunset-horizon-400p);
+}
+
+@theme inline {
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)